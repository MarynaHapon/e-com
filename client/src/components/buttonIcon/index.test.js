// Core
import React from 'react';
import { shallow } from 'enzyme';

// Component
import { ButtonIcon } from './index';

it('expect to render ButtonIcon component', () => {
  expect(shallow(
    <ButtonIcon
      type='button'
      variant='primary'
      className='collectionItemButton'
      onClick={() => {}}
      icon={<svg />}
    />
  )).toMatchSnapshot();
});
