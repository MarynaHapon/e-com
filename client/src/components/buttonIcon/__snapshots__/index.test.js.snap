// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ButtonIcon component 1`] = `
<button
  className="collectionItemButton buttonIcon primary"
  onClick={[Function]}
  type="button"
>
  <svg />
</button>
`;
